site/gatsby-site/cypress/e2e/integration/admin.cy.js

```diff
@@ -5,7 +5,7 @@ const { gql } = require('@apollo/client');
 describe('Admin', () => {
   const baseUrl = '/admin';
 
-  it('Should show not enough permissions message', () => {
+  it.skip('Should show not enough permissions message', () => {
```

Comment:
nit: Please leave a comment on why it's skipped. It should be very temporary but you know how that goes...
e.g.
"// TODO(Issue #2272): Temporarily skip flaky test."